src/renderer/apis/commands.ts

```diff
@@ -26,10 +26,10 @@ interface CommandsAndSection {
   commands: Map<string, AnyRepluggedCommand>;
 }
 
-void waitForModule<typeof User>(filters.bySource(".isStaffPersonal=")).then((User) => {
+void waitForModule<typeof User>(filters.bySource('"isStaffPersonal",')).then((User) => {
   RepluggedUser = new User({
     avatar: "replugged",
-    id: "replugged",
+    id: "69",
```

Comment:
This change is no longer required. I got it working just by replacing the source match.

Reply:
should i close the pr then?

Reply:
No? The source match change is still required.

Reply:
Yes, I also saw that, but you have to consider that was quite a while ago. I tested it *today* without replacing the `id` and it worked fine. However, I didn't test clicking the fake user. Is that where the issue lies?

Reply:
yes